### DISCUSSION TITLE
Leistungsanzeige friert ein - SAE mit go-e Charger und SMA SHM2.0

### DISCUSSION THREAD

**Discussion by wolliboy** [camueller/SmartApplianceEnabler#183] [Installation und Betrieb]
Hallo.
Ein kleines Problem habe ich derzeit:
SAE v1.6.18
go-e charger homefix 11kW Hardware v2, Firmware 041.0
SMA SHM2.0

Ich habe den go-e Charger mittels SAE in das Sunny Portal aufgenommen. Funktioniert soweit gut.
Problem:
Wenn das Auto lädt und man losfahren möchte, ziehe ich einfach den Stecker aus dem Auto und fahre los.
Dadurch wird der Zählerstand in der Wallbox $.dws (das ist der Zwischenzähler) wieder auf 0 gesetzt, was den SAE wohl einfrieren lässt, denn es wird im Sunny Portal für diesen Verbraucher die ganze Zeit eine Leistung angezeigt obwohl das Auto gar nicht mehr angeschlossen ist.
Workaround: Nicht einfach den Stecker ziehen, sondern das Laden über die App der Wallbox beenden, dann erst den Stecker ziehen. Aber das ist natürlich etwas umständlich, zumal ich das laden ja auch wieder händisch starten muss, wenn das Auto wieder geladen werden soll.
Es scheint mir als hätte der SAE ein Problem damit, dass der Zählerstand auf 0 gesetzt wird.
Den Gesamtzähler kann ich nicht verwenden, weil der nur ganzzahlige Werte ausgibt, keine Kommastallen, daher zu ungenau.
Und das über den Leistungswert zu machen ist wohl auch zu ungenau.

Gibt es Abhilfe? Anbei noch meine Config und ein Screenshot aus der SMA App.

Grüße
Wolfgang
![go-e charger im SAE_1](https://user-images.githubusercontent.com/98582148/151515731-92305ba4-4690-4895-a953-ea2cdf00be34.PNG)
![go-e charger im SAE_2](https://user-images.githubusercontent.com/98582148/151515785-bd54ebd5-4656-4830-927c-35de0679d2ea.PNG)
![InkedScreenshot_20220128-081043_SMA Energy_LI](https://user-images.githubusercontent.com/98582148/151516291-b3b66e10-7709-4346-9123-db46a269985c.jpg)

**Comment by camueller** [comment]
Kannst Du bitte mal ein Log verfügbar machen (Mail mit Link an mich)? Man sollte natürlich schon einfach den Stecker ziehen können ohne Fehhler im SAE.

**Comment by wolliboy** [comment]


**Reply by camueller** [reply]
Nein

**Comment by camueller** [comment]
Indem Du die Log-Datei mit dem Fehlverhalten zu Google-Drive oder Dropbox hochlädst und mir per Email den Link dazu schickst. Wenn Du den Link hier postest, kann jeder das Log-File sehen, was aus Datenschutzgründen nicht ideal ist :grin:

**Reply by wolliboy** [reply]
Ja ok, an welche Email soll ich es schicken? Irgendwie habe ich das hier noch nicht kapiert....? Sorry.

**Reply by camueller** [reply]
Schick den Link einfach an axel.mueller@avanux.de

**Reply by 1800Power** [reply]
Hallo, 
bei meiner fast identischen Konfiguration (Lader ist ein V3) ist die Seriennummer 6-stellig (ich hatte z.B. im Home-Assistant mit der 4-stelligen Nummer keinen Zugang) und anstatt "Sonstige" unter Typ steht bei mir "Elektroauto-Ladegerät". Das Feld "Daten =Zählerstand" habe ich leer gelassen.
Mein Stecker geht erst aus dem Auto, wenn ich den Ladevorgang beende, entweder mit dem SAE oder mit der go-e-APP.
Gruß Frank

**Reply by wolliboy** [reply]
Neue Erkenntnis meinerseits:
Mein Schwiegervater hat seit kurzem auch einen go-e charger, jedoch natürlich die neuere Variante v3. (Die v2 wird ja nicht mehr ausgeliefert).
Da verhält sich der Energiezähler anders:
Wird das Auto während des Ladens ausgesteckert, so bleibt der Zählerstand stehen. Also ich glaube das es somit in der Hinsicht bei der v3 Wallbox keine Schwierigkeiten gibt. Bei der v2 wird der Zähler direkt auf 0 gesetzt.
Das Problem was ich anfangs geschildert habe betrifft also wohl nur die v2.

Von Axel habe jetzt auch noch keine Rückmeldung...

Wäre nun die Frage ob man am SAE da was ändern kann, was ich mir eher vorstellen kann, als das die Fa. go-e da entsprechend die Firmware der v2 ändert um das gleiche Verhalten wie bei der v3 zu haben....?

Grüße
Wolfgang

**Comment by camueller** [comment]
In Deinen Logs finden sich 2 HTTP-Fehler beim Versuch, den Zählerwert des go-eCharger auszulesen:
- 2022-01-28 15:36:58,356
- 2022-01-29 17:42:57,851

Mit der 1.6.18 kann ich reproduzieren, dass nach einem HTTP-Fehler keine weiteren Abfragen erfolgen.
Mit den in der Zwischenzeit gemachten Änderungen läuft der SAE aber ganz normal weiter trotz des Fehlers.
Insofern sollte das Problem bei Dir mit der nächsten Version nicht mehr auftreten.

**Reply by wolliboy** [reply]
Moin.
Ich habe nun den SAE 2.0.2 am laufen und habe das in den letzten Tagen beobachtet.
Sieht gut aus, also scheint jetzt wie gewünscht zu funktionieren.
Mir ist nur aufgefallen, dass der ganze SAE ab und zu eingefroren / abgestürzt war. Ich musste dann den SAE Service neu starten und auch mal den kompletten Raspberry neu starten. 
 Ich habe dann mal den Deckel von meinem Raspberry geöffnet, seitdem ist es nicht mehr passiert, also könnte ein thermisches Problem gewesen sein.
Ich lasse es mal so laufen und berichte wenn es Probleme gibt.
Vielen Dank schon mal für die Hilfe.

Grüße
